tests/Adapter/DoctrineItemsTest.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace tests\Libero\ContentApiBundle\Adapter;
+
+use Doctrine\DBAL\Connection;
+use Doctrine\DBAL\DriverManager;
+use Libero\ContentApiBundle\Adapter\DoctrineItems;
+use Libero\ContentApiBundle\Model\Items;
+use Libero\ContentApiBundle\Test\ItemsTestCase;
+use function md5;
+use function uniqid;
+
+/**
+ * @group database
+ */
+final class DoctrineItemsTest extends ItemsTestCase
+{
+    /** @var Connection */
+    private $connection;
+
+    protected function createItems() : Items
+    {
+        $this->connection = DriverManager::getConnection(['url' => $_SERVER['DB_URI'] ?? 'sqlite://memory']);
+
+        $items = new DoctrineItems($this->connection, 'test_'.md5(uniqid('', true)));
+
+        $this->connection->beginTransaction();
```

Comment:
uh, relies on nested transactions; compatible with all the databases involved?

Reply:
[Doctrine doesn't do real nested transactions.](https://www.doctrine-project.org/projects/doctrine-dbal/en/2.8/reference/transactions.html#transaction-nesting)

Other option is to have an `@after` step that removes the tables (presumably won't happen on a crash, but 🤷‍♂️ ).

Reply:
Yes, [Table Truncation Teardown](http://xunitpatterns.com/Table%20Truncation%20Teardown.html) could be more reliable; it doesn't need to modify the schema is all the data can be removed.